test/e2e/cli/cli_test.go

```diff
@@ -120,8 +123,39 @@ var _ = Describe("cli operation", func() {
 
 	})
 
+	Context("certificate generation per user", func() {
+		It("should have worked, and we can have a certificate", func() {
+			out, err := harness.CLI("certificate", "request", "-n", RandStringBytes(5))
+			Expect(err).ToNot(HaveOccurred())
+			Expect(out).To(ContainSubstring("certificate is ready"))
+		})
+	})
+
+	Context("list devices", func() {
+		It("Should let you list devices", func() {
+			out, err := harness.CLI("get", "devices")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(out).To(ContainSubstring("Fleet/default"))
+		})
+	})
+
+	Context("list fleets", func() {
+		It("Should let you list fleets", func() {
+			out, err := harness.CLI("get", "fleets")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(out).To(ContainSubstring("e2e-test-fleet"))
+		})
+	})
 })
 
+func RandStringBytes(n int) string {
+	b := make([]byte, n)
+	for i := range b {
+		b[i] = letterBytes[rand.Int63()%int64(len(letterBytes))]
```

Comment:
nit: rand.Int63 is part of math/rand vs crypto/rand

ref: https://pkg.go.dev/math/rand#Int63n

Reply:
yes, changed in the latest version